src/pages/index.js

```diff
@@ -107,18 +131,80 @@ profileAddButton.addEventListener("click", function () {
 
 /* Functions */
 
-// Define the createCard function
-function createCard(cardData) {
-  // Select the template element and clone its contents
-  const cardInfo = new Card(cardData, "#card__template", function () {
-    openZoomPicture(cardData.link, cardData.name);
+function handleAvatarUpdate({ link }) {
+  return api.updateAvatar({ avatar: link }).then((res) => {
+    userInfo.setUserInfo({
```

Comment:
* See here https://snipboard.io/2b59Ke.jpg . The avatar submit button should be also disabled in `then`, if the input is empty 


Please, add `disableSubmit` for the avatar here